component/portal/src/main/java/org/exoplatform/portal/rest/UserRestResourcesV1.java

```diff
@@ -510,6 +510,26 @@ public Response isSuperUser() {
     return Response.ok().entity("{\"isSuperUser\":\"" + userACL.isSuperUser() + "\"}").build();
   }
 
+  @GET
+  @Path("isDelegatedAdministrator")
+  @Produces(MediaType.APPLICATION_JSON)
+  @RolesAllowed("users")
+  @ApiOperation(
+          value = "Check if current user is a delegated administrator",
+          httpMethod = "GET",
+          response = Response.class
+  )
+  @ApiResponses(
+          value = {
+                  @ApiResponse(code = 200, message = "Request fulfilled"),
+                  @ApiResponse(code = 500, message = "Internal server error due to data encoding"),
+          }
+  )
+  public Response isDelegatedAdministrator() {
+    boolean isDelegatedAdministrator = userACL.isUserInGroup("/platform/delegated") && !userACL.isUserInGroup("/platform/administrators");
```

Comment:
it is possible to put "/platform/delegated" and "/platform/administrators" as constants ?

Reply:
yes, done !